src/tests/baseservices/exceptions/unhandled/unhandledTester.cs

```diff
@@ -142,6 +149,8 @@ public static void TestEntryPoint()
             RunExternalProcess("foreign", "unhandled.dll");
             File.Delete(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "dependencytodelete.dll"));
             RunExternalProcess("missingdependency", "unhandledmissingdependency.dll");
+            if (!TestLibrary.Utilities.IsMonoRuntime && !TestLibrary.Utilities.IsNativeAot)
+                RunExternalProcess("collecteddelegate", "collecteddelegate.dll");
```

Comment:
Does this test need to run as an "unhandled exception" test or can it run as a test that just asserts the exception message?